ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/message/AbstractMessageHandler.java

```diff
@@ -56,12 +56,8 @@
     return databasesUpdated;
   }
 
-  ReplicationSpec eventOnlyReplicationSpec(Context forContext) throws SemanticException {
```

Comment:
This code was refactored by you and since then we have it. :)
However, that's good suggestion to move it in Context class. Will make the change.